dockers/docker-orchagent-cavm/copp.json

```diff
@@ -0,0 +1,9 @@
+[
```

Comment:
1. There was the idea that copp configuration will be vendor specific. Is the idea dropped now?
2. I can not find where you run swssconfig utility
3. It seems swssconfig has a bug when opens copp.json with write permissions.